### DISCUSSION TITLE
Chinese (death) numbers

### DISCUSSION THREAD

**Discussion by marcopiani** [owid/covid-19-data#2593] [General]
Hi,

it might not be OWID policy or position, but given the attention given to -- and the objective importance of -- the situation in China (particularly, Shanghai), perhaps you could consider either pulling the Chinese data (at least for deaths) or putting an extra warning about their reliability.

I have seen some people sharing the (totally unrealistic) ~zero-death graph.

According to the present charts: 2 deaths reported since January 2022, compared to 300k cases.
![image](https://user-images.githubusercontent.com/18381804/162772838-a5f7b42a-92d8-40af-b4b5-317b1d8a5fdf.png)
![image](https://user-images.githubusercontent.com/18381804/162773002-9f41c214-3e6c-417f-842d-327d05a2557a.png)

Just afraid this kind of data is going to muddle the view/understanding of the situation.

Thanks for any consideration you may give to the options suggested (or any other!)

EDIT: I am aware of the general warning you already indicate, but I think this issue goes somewhat beyond that level of "mismatch" between data and reality

Marco

**Comment by edomt** [comment]
Hi @marcopiani 

While we agree with your concerns about confirmed deaths in China, it's also the case that the data from other countries looks inaccurate or biased, so it seems that specifically highlighting the case of China wouldn't be the right solution.

By now we also hope that people are used to the fact that we show only confirmed, reported statistics and this depends on how countries report.

**Reply by marcopiani** [reply]
Hi @edomt ,

Thanks for considering the issue. I think many people have been emphasizing the unreliability of the official Chinese mortality data, and hopefully that will be enough. Your note under the title helps.

Thanks again.

Marco
